packages/core/medusa-test-utils/src/medusa-test-runner-utils/bootstrap-app.ts

```diff
@@ -19,7 +19,7 @@ async function bootstrapApp({
     expressApp: app,
   })
 
-  const PORT = await getPort()
+  const PORT = process.env.PORT ? parseInt(process.env.PORT) : await getPort()
```

Comment:
Q: would it be useful to be able to provide an optional port as part of the `medusaIntegrationTestRunner` input args? and you could pass ({ port: process.env.PORT })

Reply:
I was thinking like this you won't need to add an additional argument, and you can do
```
medusaTestRunner({env: {PORT: 3000}})
```

But I am happy to do it either way, let me know which one you prefer.

Reply:
I think we can keep it as is, just wanted your opinion :)